echopop/nwfsc_feat/load_data.py

```
@@ -44,73 +48,75 @@ def load_biological_data(
     >>> label_map = {"sex": {1: "male", 2: "female", 3: "unsexed"}}
     >>> bio_data = load_biological_data("biodata.xlsx", sheet_map, col_map, subset, label_map)
     """
-    
+
     if not biodata_filepath.exists():
         raise FileNotFoundError(f"Biological data file not found: {biodata_filepath}")
-    
+
     # Load each biological dataset
     biodata_dict = {
         dataset_name: load_single_biological_sheet(
             biodata_filepath, sheet_name, column_name_map, subset_dict
         )
         for dataset_name, sheet_name in biodata_sheet_map.items()
     }
-    
+
     # Apply label mappings if provided
     if biodata_label_map:
         biodata_dict = preprocess_biological_data(biodata_dict, biodata_label_map)
-    
+
     return biodata_dict
 
+
 def preprocess_biological_data(
-    bio_data: Dict[str, pd.DataFrame], 
-    biodata_label_map: Optional[Dict[str, Dict]] = None
+    bio_data: Dict[str, pd.DataFrame], biodata_label_map: Optional[Dict[str, Dict]] = None
 ) -> Dict[str, pd.DataFrame]:
     """
     Apply label mappings to biological data.
-    
+
     Parameters
     ----------
     bio_data : dict
         Dictionary of DataFrames containing biological data
     biodata_label_map : dict, optional
         Dictionary mapping column names to value replacement dictionaries
         Example: {"sex": {1: "male", 2: "female", 3: "unsexed"}}
-        
+
     Returns
     -------
     dict
         Dictionary of processed DataFrames
     """
     if not biodata_label_map:
         return bio_data
-        
+
```

This should error out if no `biodata_label_map` is not provided.